app/controllers/import_controller.rb

```diff
@@ -123,6 +123,22 @@ def import
               @errors << ValidationError.new(error, row_ind).message
             end
           end
+
+          cohorts = []
+          cohorts.push(cohort(row[132..134], row_ind)) if row[132..134].filter(&:present?).any?
+          cohorts.push(cohort(row[135..137], row_ind)) if row[135..137].filter(&:present?).any?
+
+          patient[:common_exposure_cohorts_attributes] = cohorts unless cohorts.empty?
+
+          # Validate using Common Exposure Cohort model validators without saving
+          cohorts.each do |cohort_data|
+            validation_cohort = CommonExposureCohort.new(cohort_data)
+            next if validation_cohort.valid?
+
+            format_model_validation_errors(validation_cohort).each do |error|
+              @errors << ValidationError.new(error, row_ind).message
+            end
```

Comment:
This doesn't seem to work for me, I don't get validation errors with clear messages, but just get a 422 error when I actually try to import.

Reply:
good catch, this should have fixed it: https://github.com/SaraAlert/SaraAlert/pull/1125/commits/237545888b608d51912da918ac3e3e6d0780ee28#diff-4f61e388997f8d92e41f66947a47b461707b615cf9fac69f1080b8254630d2ceR331